core/ir/encode_api.h

```diff
@@ -51,38 +51,52 @@ typedef enum _dr_isa_mode_t {
     DR_ISA_ARM_THUMB,         /**< Thumb (ARM T32). */
     DR_ISA_ARM_A64,           /**< ARM A64 (AArch64). */
     DR_ISA_RV64IMAFDC,        /**< RISC-V (rv64imafdc). */
+                              /**
+                               * This is a synthetic ISA that has the purpose of
```

Comment:
Wow, clang-format really indents this to line up with the < comments?  Maybe you should turn the comments above into pre-comments just to avoid this massive indentation.

Reply:
Yeah, I tried to shift it to the left, but clang-format complained.
Turned all of these into pre-comments.